.github/workflows/reduce-readme.yml

```diff
@@ -0,0 +1,34 @@
+name: Reduce README
+on:
+  push:
+    paths:
+    - '**'
+    branches: ['**']
+jobs:
```

Comment:
We also need to be able to cancel new runs of same job so please add:
```
# Only run the latest job
concurrency:
  group: '${{ github.workflow }} @ ${{ github.ref || github.run_id }}'
  cancel-in-progress: true
```

Reply:
added